src/vulkan_aliasing_2.cpp

```diff
@@ -104,11 +104,13 @@ int main(int argc, char** argv)
 
 	if (vulkan.vkAssertBuffer)
 	{
-		const uint32_t crc_child_1 = vulkan.vkAssertBuffer(vulkan.device, child_1, 0, VK_WHOLE_SIZE, "child_1 buffer");
+		uint32_t crc_child_1 = 0;
+		uint32_t crc_child_2 = 0;
+		assert(VK_SUCCESS == vulkan.vkAssertBuffer(vulkan.device, child_1, 0, VK_WHOLE_SIZE, &crc_child_1, "child_1 buffer"));
```

Comment:
Putting the Vulkan call inside an assert macro means it may be optimized away for release builds, which is wrong.